build.gradle

```diff
@@ -602,7 +603,6 @@ dependencies {
     testImplementation group: 'org.powermock', name: 'powermock-api-support', version: '2.0.2'
     testImplementation group: 'org.powermock', name: 'powermock-reflect', version: '2.0.7'
     testImplementation group: 'org.objenesis', name: 'objenesis', version: '3.0.1'
-    testImplementation group: 'org.javassist', name: 'javassist', version: '3.27.0-GA'
```

Comment:
What is the reason for removing this?

Reply:
This will cause jar hell
```
| Exception in thread "main" java.lang.IllegalStateException: jar hell!
| class: javassist.ByteArrayClassPath$1
| jar1: /local/home/ylwu/.gradle/caches/modules-2/files-2.1/org.opensearch/opensearch-job-scheduler-spi/1.3.0.0-SNAPSHOT/ff18d5a98e4a09bee254d306a165d63f25908f60/opensearch-job-scheduler-spi-1.3.0.0-SNAPSHOT.jar
| jar2: /local/home/ylwu/.gradle/caches/modules-2/files-2.1/org.javassist/javassist/3.27.0-GA/f63e6aa899e15eca8fdaa402a79af4c417252213/javassist-3.27.0-GA.jar
```